taf/cli.py

```diff
@@ -18,5 +19,40 @@ def add_target_file(repo_path, file_path, keystore_path, targets_role):
     tuf_repo.add_existing_target(file_path)
     tuf_repo.write_roles_metadata(targets_role, keystore_path, True)
 
+@cli.command()
+@click.option('--repo-path', help='Authentication repository\'s path')
+@click.option('--keystore-path', help='Path of the keystore file')
+@click.option('--targets-role', default='targets', help='Targets metadata file to be updated')
+def add_targets(repo_path, keystore_path, targets_role):
+  targets_path = os.path.join(repo_path, 'targets')
```

Comment:
`CLI` should be just a wrapper around our core logic. We should wrap below lines in a core function and call it in one line (like you did for `taf update` CLI).
I can change this once I start working on TAF Management app.

Reply:
I agree. We could do something similar to how we handle build in platform. This was just something I quickly put together in order to be able to generate initial auth repository.